### ISSUE TITLE
Any idea what this error mean?

### ISSUE THREAD

**Issue by drveera** [zkutalik/ssimp_software#98]
<img width="1425" alt="Screen Shot 2019-05-02 at 19 02 50" src="https://user-images.githubusercontent.com/8498890/57093079-3b652e80-6d0d-11e9-98fd-9c16383fdeef.png">
Hi I am trying to use the latest version. I have no issues with the old version 0.1, but the latest version is throwing this error. 
I couldn't understand the meaning of the error message. I'm happy to answer any further questions you'll have in relation to this error message. 
Thanks :)
regards
Veera

**Comment by sinarueeger** [comment]
Hi Veera, 
Could you provide us with the full error message, the exact command used, and the first few lines of the input files, so we can try to reproduce your error?

@zkutalik @aaron-mcdaid-zalando

**Comment by MarioGuCBMR** [comment]
Hi all,

I have found the same error. I have the latest version and I am using the linux compiled version. The commands are the following:

./ssimp --gwas gwas/small.random.txt --ref home/zlc436/reference_panels/1000genomes/ALL.chr{CHRM}.phase3_shapeit2_mvncall_integrated_v5a.20130502.genotypes.vcf.gz --out output.txt --sample.names /home/zlc436/reference_panels/1000genomes/integrated_call_samples_v3.20130502.ALL.panel/sample/super_pop=EUR

It seems that I could only download the 1000G data. Any other commands, with or without flags, shows me this error. 

Thanks in advance for your help!

**Comment by zkutalik** [comment]
Hi Mario!

Could you please share the first couple of lines of your input file so that we can see the chr/pos in your small.random.txt summary stat file along with the exact error message? It seems your positions do not match any build...

Thanks!

**Comment by MarioGuCBMR** [comment]
Hi,

Thanks for your fast answer. Here you have it:


MarkerName Allele1 Allele2 Z
rs5992801 T C 0.02625569846962102
rs1123031 T C 0.11867322079278599
rs5992111 T C 0.051311166629648186

It is the file I downloaded along with ssIMP. Also, I have tried to use Fasting Insulin summary statistics with the same header, but it does not work. I have seen the headers as A1 and A2 in some of your examples. Could it be that? I haven't tried it myself yet.

Cheers,

Mario


________________________________________
De: zkutalik [notifications@github.com]
Enviado: jueves, 7 de mayo de 2020 9:44
Para: zkutalik/ssimp_software
Cc: Mario García Ureña; Comment
Asunto: Re: [zkutalik/ssimp_software] Any idea what this error mean? (#98)

Hi Mario!

Could you please share the first couple of lines of your input file so that we can see the chr/pos in your small.random.txt summary stat file along with the exact error message? It seems your positions do not match any build...

Thanks!

**Comment by zkutalik** [comment]
Sorry, just to make sure, you got the exact same error message as drveera? Even iIdentical counts in each build category (0,47,14,0,899,237)?

**Comment by MarioGuCBMR** [comment]
That is a good question. I forward you here the error I get:

terminate called after throwing an instance of 'std::runtime_error'
  what():  locale::facet::_S_create_c_locale name not valid
Aborted

My command is the following:

./ssimp --gwas gwas/small.random.txt --ref home/zlc436/reference_panels/1000genomes/ALL.chr{CHRM}.phase3_shapeit2_mvncall_integrated_v5a.20130502.genotypes.vcf.gz --out output.txt --sample.names /home/zlc436/reference_panels/1000genomes/integrated_call_samples_v3.20130502.ALL.panel/sample/super_pop=EUR

Cheers,

Mario

________________________________________
De: zkutalik [notifications@github.com]
Enviado: viernes, 8 de mayo de 2020 9:56
Para: zkutalik/ssimp_software
Cc: Mario García Ureña; Comment
Asunto: Re: [zkutalik/ssimp_software] Any idea what this error mean? (#98)

Sorry, just to make sure, you got the exact same error message as drveera? Even iIdentical counts in each build category (0,47,14,0,899,237)?

**Comment by zkutalik** [comment]
Could you please share the entire log file, just to see at what point this error occurred?

Thanks

Zoltan

**Comment by zkutalik** [comment]
Also could you please let me know what your computer gives if you type "locale"? Often "export LC_ALL=C" could solve this problem.

**Comment by MarioGuCBMR** [comment]
Hi, 

When I run locale, I get the following:

LANG=en_US.UTF-8
LC_CTYPE="en_US.UTF-8"
LC_NUMERIC="en_US.UTF-8"
LC_TIME="en_US.UTF-8"
LC_COLLATE="en_US.UTF-8"
LC_MONETARY="en_US.UTF-8"
LC_MESSAGES="en_US.UTF-8"
LC_PAPER="en_US.UTF-8"
LC_NAME="en_US.UTF-8"
LC_ADDRESS="en_US.UTF-8"
LC_TELEPHONE="en_US.UTF-8"
LC_MEASUREMENT="en_US.UTF-8"
LC_IDENTIFICATION="en_US.UTF-8"
LC_ALL=

I guess the problem might be solved by rewriting it in the way you said it right?

Mario
________________________________________
De: zkutalik [notifications@github.com]
Enviado: viernes, 8 de mayo de 2020 13:01
Para: zkutalik/ssimp_software
Cc: Mario García Ureña; Comment
Asunto: Re: [zkutalik/ssimp_software] Any idea what this error mean? (#98)

Also could you please let me know what your computer gives if you type "locale"? Often "export LC_ALL=C" could solve this problem.

**Comment by zkutalik** [comment]
Yes, that command would be the first to try. Some others (https://github.com/potree/PotreeConverter/issues/281) suggest executing "export LC_ALL=C; unset LANGUAGE" before running our tool [or add these to your ".profile" file in your home directory to avoid typing it all the time].

This error has to do with your locale settings, in some cases it needs reinstallation of locale. (https://community.appdynamics.com/t5/Dynamic-Languages-Node-JS-Python/locale-facet-S-create-c-locale-name-not-valid/td-p/33693).

Please have a look around for other solutions in the internet.

**Comment by MarioGuCBMR** [comment]
Dear Zoltan,

Thank you so much, I think that with this I have a good grasp of the problem. Thank you again for you help and your quick answers.

Cheers,

Mario
________________________________________
De: zkutalik [notifications@github.com]
Enviado: viernes, 8 de mayo de 2020 13:17
Para: zkutalik/ssimp_software
Cc: Mario García Ureña; Comment
Asunto: Re: [zkutalik/ssimp_software] Any idea what this error mean? (#98)

Yes, that command would be the first to try. Some others (potree/PotreeConverter#281<https://eur02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fpotree%2FPotreeConverter%2Fissues%2F281&data=02%7C01%7Cmario.garcia.urena%40sund.ku.dk%7C1941435f8c49410b420c08d7f34168f6%7Ca3927f91cda14696af898c9f1ceffa91%7C0%7C0%7C637245334566548904&sdata=DuqybqIFQO4iamskMxq5baA3Ql77Tkj%2F4wZlD9Xga4Y%3D&reserved=0>) suggest executing "export LC_ALL=C; unset LANGUAGE" before running our tool [or add these to your ".profile" file in your home directory to avoid typing it all the time].

This error has to do with your locale settings, in some cases it needs reinstallation of locale. (https://community.appdynamics.com/t5/Dynamic-Languages-Node-JS-Python/locale-facet-S-create-c-locale-name-not-valid/td-p/33693<https://eur02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fcommunity.appdynamics.com%2Ft5%2FDynamic-Languages-Node-JS-Python%2Flocale-facet-S-create-c-locale-name-not-valid%2Ftd-p%2F33693&data=02%7C01%7Cmario.garcia.urena%40sund.ku.dk%7C1941435f8c49410b420c08d7f34168f6%7Ca3927f91cda14696af898c9f1ceffa91%7C0%7C0%7C637245334566548904&sdata=dCoU5J%2BeVBPslsCwCo8CY9hZPGgBb2YwtRdFK3XIpjA%3D&reserved=0>).

Please have a look around for other solutions in the internet.

**Comment by zkutalik** [comment]
Hope one of these suggestions will help sorting your problem! Please let us know if it still persists after trying all those tips.

Best

Zoltan
